.storybook/helpers.ts

```diff
@@ -14,3 +17,37 @@ export const boolean = (prop, value, standalone = true) => {
   const attrValue = standalone ? "" : `="${knob}"`;
   return `${propValue}${attrValue}`;
 };
+
+export interface Story {
+  (): string;
+  decorators?: ((Story: Story) => DocumentFragment)[];
+}
+
+export const setKnobScript = ({ story, knob, value }: { story: string; knob: string; value: string }) => {
```

Comment:
nitpick: perhaps the name could be `setKnobURL`.  The word `script` makes it sound like it adds a `<script>` tag to the page.